src/main/java/modtrekt/logic/commands/DoneModuleCommand.java

```diff
@@ -62,6 +66,29 @@ public CommandResult execute(Model model) throws CommandException {
 
         // Done the module.
         model.setModule(target, target.done());
+        // Sum up credits.
+        ObservableList<Module> modules = model.getModuleList().getModuleList();
+        refresh(modules);
         return new CommandResult("Marked module as done.");
     }
+
+    /**
+     * Counting the number of MC Completed.
+     * @param modules module list.
+     */
+    public static void refresh(ObservableList<Module> modules) {
+        // Sum up credits.
+        Stream<Module> doneModules = modules.stream().filter(x -> x.isDone());
+        int totalCreditsDone = doneModules.reduce(0, (creditsResult, mod) ->
+                creditsResult + mod.getCredits().getIntValue(), Integer::sum);
+        totalCredits = totalCreditsDone;
```

Comment:
This can be simplified significantly:

```suggestion
        totalCredits = modules.stream()
                .filter(Module::isDone)
                .mapToInt(mod -> mod.getCredits().getIntValue())
                .sum();
```